src/wilson/dom.cljs

```diff
@@ -63,10 +72,43 @@
           (for [row rows]
             (into [:tr (row->attrs row)]
                   (for [k ks]
-                    [:td (k row)]))))])
+                    (let [td-data (k row)]
+                      [:td (data->hiccup td-data)])))))])
   ([ks rows]
    (table ks rows {})))
 
+
+(defn sorted-table
+  ([ks rows opts]
```

Comment:
(also potentially there should be an arity that stores it in the reagent state instead of doing this; possibly with an unambiguous name/prefix/id)